zipline/sources/test_source.py

```diff
@@ -153,10 +150,21 @@ def __init__(self, *args, **kwargs):
                 self.event_list[1].dt - self.event_list[0].dt)
             self.concurrent = kwargs.get('concurrent', False)
 
+            self.identifiers = kwargs.get(
+                'sids',
+                np.unique([event.sid for event in self.event_list]).tolist())
```

Comment:
If this just needs to be an iterable, this can be written as `set(event.sid for event in self.event_list)`.